mhkit/power/quality/calc_alpha0_freq.m

```diff
@@ -0,0 +1,87 @@
+function [alpha0,freq] = calc_alpha0_freq(u_m,method,methodopts)
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%   Calculates the fundamental frequency of the measured voltage u_m(t)
+%   using Zero-Crossing-Detection (default) or Short-Time Fourier 
+%   Transform (STFT).
+%   
+% Parameters
+% -----------
+%   u_m: struct()
+%       .time: time for each time step; 
+%       .data: array of size (ntime), measured voltage (V) with instantaneous values u_m(t).
+%   
+%   method: string (case-insensitive)
+%       'zcd' or 'stft', default is 'zcd'.
+%   
+%   methodopts: cell array 
+%       Name-value arguments for STFT, e.g.,
+%       opts = {'Window',rectwin(M),'OverlapLength',L,
+%               'FFTLength',Nfrq,'FrequencyRange','onesided'}
+%       if using 'ZCD' method, can set to {}.

```

Comment:
Is this the default value? If so, make this an optional argument and indicate that it is required for stft method. Will also be useful to have a check in the code and an error thrown if it is not set for soft

Reply:
Since this method option is crucial and really dependent on some prior knowledge of data, it'd be better to be without a default choice for this input. Sorry that this is misleading, I will change the code doc.